src/AdaptivePopover.tsx

```diff
@@ -230,6 +230,11 @@ export default class AdaptivePopover extends Component<AdaptivePopoverProps, Ada
     count = 0;
     do {
       rect = await getRectForRef(fromRef);
+      // console.log('rect', rect)
```

Comment:
Please remove this comment

Reply:
https://github.com/SteffeyDev/react-native-popover-view/pull/147/commits/09bee705372526786209afb104c972938cf69ac1